Common/src/main/resources/defaultresources/excavated_variants/excavated_variants/variants/deepercaves.json5

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2023 Luke Bemish and contributors
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+*/
+
+{
+  mod_id: 'cavesanddepths',
+  provided_stones: [
+    {
+      id: 'chert',
+      lang: {
+        en_us: 'Chert',
+      },
+      block_id: 'cavesanddepths:chert',
+      types: [
+        'stone',
+      ],
+    },
+    {
+      id: 'chalkstone',
+      lang: {
+        en_us: 'Chalkstone',
+      },
+      block_id: 'cavesanddepths:chalkstone',
+      types: [
+        'stone',
+      ],
+    },
+    {
+      id: 'mudstone',
+      lang: {
+        en_us: 'Mudstone',
+      },
+      block_id: 'cavesanddepths:mudstone',
+      types: [
+        'stone',
+      ],
+    },
+    {
+      id: 'slate',
+      lang: {
+        en_us: 'Slate',
+      },
+      block_id: 'cavesanddepths:slate',
+      types: [
+		'stone',
+      ],
+    },
+  ],
+  provided_ores: [
+	{
+      id: 'slate_coal_ore',
+      orename: 'coal_ore',
+      lang: {
+		en_us: 'Coal Ore',
+      },
+      block_id: [
+        'cavesanddepths:slate_coal_ore',
+      ],
+      types: [
```

Comment:
I feel like these ought to have at least one type, to actually generate any variants?

Reply:
No, the point is to blacklist it. They're just reskins of vanilla ores, and I didn't want the mod to generate duplicate slate versions of each one since it's already covered by vanilla ores existing.

Reply:
What, no. It _won't_ generate duplicate versions if you use the same ore ID, cause the entries from this and the entries from the vanilla coal ore file will get merged. The issue is that you're using the ID `slate_coal_ore` for some reason so it thinks its another ore, cause, well, you _told_ it that it is.

Reply:
The problem is that doing so makes it entirely remove the Slate Ores. That's why I set up this blacklist system. It's the only way to get the mod to keep the Slate Ores while not adding duplicates for them.
![image](https://github.com/user-attachments/assets/e971d136-3fcc-4948-b02d-be20d9b22cf1)

Reply:
Had issues with it before (that's why I made the workaround), but it seems to be working now. Finally lol.